### PR TITLE
Simplifying and linking to new efforts

### PR DESCRIPTION
**Pull request by mbbroberg**
Relates to https://github.com/sensu-plugins/community/issues/58

Simplifying the messaging here so people know what to expect and the basics of getting involved. Would love to get people filtering into GitHub to help on the future.

Pic of new index: 
screen shot 2017-10-06 at 2 33 23 pm

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -35,4 +35,4 @@ collections:
   docs:
     output: true
 
-exclude_repos: ["sensu-plugins-feature-requests", "GIR", "Kryten", "documentation", "sensu-plugin", "sensu-plugin-spec", "tom_servo", "sensu-plugins.github.io", "mulder", "scully" ]
+exclude_repos: ["community", "tools", "sensu-plugins-feature-requests", "GIR", "Kryten", "documentation", "sensu-plugin", "sensu-plugin-spec", "tom_servo", "sensu-plugins.github.io", "mulder", "scully" ]
```
Comment (majormoses) [inline _config.yml]: we should add `maintainers-governance`. Can we split this up to me one per line to make SCM changes easier to spot? Also not sure why `sensu-plugin-spec` was ignored but we should look at that.

File: _config.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 # Site settings
 title: Sensu Plugins
 
-support_email: support@sensuapp.org
-irc: irc.freenode#sensu
+support_email: community@sensu.io
```
Comment (majormoses) [inline _config.yml:4]: should the support email really be community? I feel like they should probably be kept seperate for at least SLA tracking purposes. Totally not my call nor do I really care beyond trying to help thinking about it.

File: _docs/FAQ.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ Tests are coming, that is one of the core requirements for stability and product
 
 ### How do I use this handler, deploy this check?
 
-The readme will contain sample configs and commands.  Many of the plugins also have a header that contains specific details.  If this is lacking ask in IRC or on the mailing list and someone will be able to assist you.
+The readme will contain sample configs and commands.  Many of the plugins also have a header that contains specific details.  If this is lacking ask in Slack or on the mailing list and someone will be able to assist you.
```
Comment (majormoses) [inline _docs/FAQ.md]: [Slack](https://slack.sensu.io)

File: _config.yml [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+slack: slack.sensu-plugins.io
```
Comment (majormoses) [inline _config.yml]: should it be slack.sensu.io ? Just trying to be consistent. I see that one used below.

File: _config.yml (hunk 1 above)
Comment (mbbroberg) [reply _config.yml]: @majormoses I realize I could delete this line since this PR also removes the plugin listing code that doesn't work.

Comment (majormoses) [reply _config.yml]: ok, makes sense.

File: _config.yml (hunk 2 above)
Comment (mbbroberg) [reply _config.yml:4]: It's a tough one. Team Sensu support should not have a direct link from this community site - it's out of scope of their work. I think of support on this page as a request to the community team at the company. The only thing I'm sure of is it shouldn't be the SLA'ed email of our customer support.

File: _config.yml (hunk 4 above)
Comment (mbbroberg) [reply _config.yml]: 🤷‍♂️  they both work.

Comment (mbbroberg) [reply _config.yml]: Fixed.

File: _config.yml (hunk 2 above)
Comment (mbbroberg) [reply _config.yml:4]: Leaving as is so Community team can monitor community emails and not expect our customer support team to receive community requests.

File: _docs/FAQ.md (hunk 3 above)
Comment (mbbroberg) [reply _docs/FAQ.md]: Fixed.

**Comment by mbbroberg** [comment]
I'll give this a little more time for comments then move to merge. It's a starting point!

**Pull request merged**